internal/services/engines/jvm/rules.go

```diff
@@ -459,10 +459,10 @@ func NewBase64Decode() *text.Rule {
 			Severity:    severities.Low.ToString(),
 			Confidence:  confidence.Low.ToString(),
 		},
-		Type: text.AndMatch,
+		Type: text.OrMatch,
 		Expressions: []*regexp.Regexp{
 			regexp.MustCompile(`android.util.Base64`),
-			regexp.MustCompile(`.decode`),
+			regexp.MustCompile(`.decode\(`),
```

Comment:
maybe you can change to
```suggestion
			regexp.MustCompile(`\.decode\(`),
```
in order not to have too many false positives, maybe it is better to make it very clear that you will need to access the `encode` or `decode` method

Reply:
Done.